roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+import search_journalctl
+
+from time import time
+
+
+def canned_search_journalctl(get_log_input=None):
+    """Create a search_journalctl object with canned get_log_input method"""
+    module = search_journalctl
+    if get_log_input:
+        module._get_log_input = get_log_input
+    return module
+
+
+def get_timestamp_microseconds(modifier=0):
+    return str(time() + modifier).split(".")[0] + "000000"
+
+
+def create_test_log_object(stamp, msg):
+    return '{{"__REALTIME_TIMESTAMP": "{}", "MESSAGE": "{}"}}'.format(stamp, msg)
+
+
+@pytest.mark.parametrize('stamp_seconds,stamp_limit_seconds,is_too_old', [
+    (1496341364.241717, time() - 60 * 60, True),
+    (float(time()), time() - 60 * 60, False),
+    (float(time() * 0.5), time() - 10, True),
+])
+def test_stamp_too_old(stamp_seconds, stamp_limit_seconds, is_too_old):
+    assert search_journalctl.stamp_too_old(stamp_seconds, stamp_limit_seconds) == is_too_old
```

Comment:
This function is really trivial, all it does is compare two numbers... If we are bothering to have it defined and testing it, then we can test it with simple constant input, doesn't need calls to `time()`. We're essentially testing the `<` operator...

```python
import operator

stamp_too_old = operator.lt
```

:smile: